internal/cluster/validator.go

```diff
@@ -262,6 +266,8 @@ func (v *clusterValidator) printIsNetworkTypeValid(context *clusterPreprocessCon
 		}
 		if doesClusterNetworksMatchNetworkType(context.cluster) {
 			return "The cluster is configured with IPv6 which is not supported by OpenShiftSDN; use OVNKubernetes instead"
+		} else if doesAvailabilityModeMatchNetworkType(context.cluster) {
```

Comment:
I don't think we should block it, in case some reason we do block it, 
note that users can use the API to set whatever network type they like e.g. Calico, Cilium, Flannel etc.
it doesn't even have to be `OpenShiftSDN`

Reply:
we should block OpenShiftSDN, default to OVN, all the rest are allowed